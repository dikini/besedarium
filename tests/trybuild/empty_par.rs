use playground::*;

type EmptyPar = tpar!(Http;);
