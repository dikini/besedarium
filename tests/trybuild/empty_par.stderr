error: unexpected end of macro invocation
 --> tests/trybuild/empty_par.rs:3:28
  |
3 | type EmptyPar = tpar!(Http;);
  |                            ^ missing tokens in macro arguments
  |
note: while trying to match meta-variable `$branch:ty`
 --> src/lib.rs
  |
  |     ($io:ty; $($branch:ty),+ $(,)?) => {
  |                ^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/trybuild/empty_par.rs:3:30
  |
3 | type EmptyPar = tpar!(Http;);
  |                              ^ consider adding a `main` function to `$DIR/tests/trybuild/empty_par.rs`
